aiozipkin/span.py

```diff
@@ -66,6 +66,10 @@ def kind(self: T, span_kind: str) -> T:
     def name(self: T, span_name: str) -> T:
         pass  # pragma: no cover
 
+    @abstractmethod
+    def new_child(self: T, name: str = None, kind: str = None) -> T:
```

Comment:
should be OptStrt or Optional[str], since name or kind could be None

Reply:
done